### PR TITLE
refactor workshop doc

### PR DESCRIPTION
**Pull request by dengmingtong**
Issue #, if available:

Description of changes:
[Preview](https://dengmingtong.github.io/service-workbench-on-aws-cn/workshop/workshop_guide)
- Refactor workshop doc
- Fix some links to en docs from cn docs
- Fix the account creation document

Checklist:

- [ yes] Have you successfully deployed to an AWS account with your changes?
- [ yes] Have you successfully deployed to an AWS China account with your changes?
- [ ] Have you written new tests for your core changes, as applicable?
- [ ] Have you successfully tested with your changes locally?
- [ ] If new dependencies have been added, have they been pinned to specific versions?
- [ ] Is this change also required on the AWS Solution version?
- [ ] Have you updated openapi.yaml if you made updates to API definition (including add, delete or update parameter and request data schema)?
- [ ] If you had to run manual tests, have you considered automating those tests by adding them to [end-to-end tests](../main/end-to-end-tests/README.md)?
- [ ] If you are updating the changelog and vending out a new release, have you updated versionNumber and versionDate in [.defaults.yml](../main/config/settings/.defaults.yml)

AS review ticket id:

By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.

### CODE REVIEW

File: docs/docs/workshop/workshop_guide.md [hunk 1]
```diff
@@ -4,143 +4,232 @@ title: 动手实践
...
+- 它集成了多个AWS 服务，例如 Amazon CloudFront、AWS Lambda、AWS Step Functions 等。您可以使用 Service Workbench 创建自定义的研究环境模板并与其他组织共享这些模板。
+- 同时，为了控制研究成本，Service Workbench 已与 AWS Cost Explorer 和 AWS Budgets 集成。您可以快速访问仪表板（dashboard)，参照预算查看您当前 AWS 服务的使用成本。
+- Service Workbench 已与第三方服务提供商集成，以便在需要时启用。
+- Service Workbench 还可以模拟本地环境中用户已知的概念，从而更轻松地切换到云技术。
+
 Service Workbench on AWS 的详细介绍请参考[Service Workbench](/zh/introduction)
 
-该手动实践默认在 **cn-northwest-1** region 进行。
+参考下面的架构图了解 Service Workbench 方案的具体架构：
+[![](img/SWB_Architecture.png)](img/SWB_Architecture.png)
```
Comment (zxkane) [inline docs/docs/workshop/workshop_guide.md:18]: KMS icon looks like it's stale

**Review by zxkane — CHANGES_REQUESTED** [review]

**Comment by ghost** [comment]
[前提条件](https://dengmingtong.github.io/service-workbench-on-aws-cn/workshop/workshop_guide/#%E5%89%8D%E6%8F%90%E6%9D%A1%E4%BB%B6)这里，有一句：
> 确保账户在 cn-northwest-1 或 ap-east-1 区域至少可以创建一个VPC

个人认为改成
> 确保账户在 cn-northwest-1 或 ap-east-1 区域至少创建一个VPC

更好一些。

**Comment by ghost** [comment]
[部署 service workbench](https://dengmingtong.github.io/service-workbench-on-aws-cn/workshop/workshop_guide/#%E9%83%A8%E7%BD%B2-service-workbench)这里有一句：
> 编辑 ${STAGE_NAME}.yml 文件,设定如下配置

个人认为对于不熟悉命令行的用户，可能理解不了${STAGE_NAME.yml}就是dev.yml这回事（这确实也是第一次dry-run时遇到过的问题）。  

所以我们可以强调一下，比如改成
> 编辑 ${STAGE_NAME}.yml （即dev.yml）文件,设定如下配置

或者直接改成
> 编辑 dev.yml 文件,设定如下配置

前面有对Stage的说明，Stage是Service Workbench一个比较重要的概念，我觉得还是应该保留。
来做workshop的人应该是有一定技术基础的。

**Comment by ghost** [comment]
[添加AWS账户](https://dengmingtong.github.io/service-workbench-on-aws-cn/zh/user_guide/sidebar/admin/accounts/aws_accounts/invite_member_account/#%E6%B7%BB%E5%8A%A0-aws-%E8%B4%A6%E6%88%B7)这里，有一句：
> 6. 在 **外部 ID** 字段中输入 AWS 账户的外部 ID，譬如`workbench`（这确实也是第一次dry-run时遇到过的问题）。

所以可以改成比如下面这样：
> 6. 在 **外部 ID** 字段中为该 AWS 账户 输入一个自定义的外部 ID，譬如`workbench`

个人认为这里可以说明一下“外部ID”的含义，或者至少说明一下这个字段是自定义的。

好建议，改成： “在 **外部 ID** 字段中为该 AWS 账户 输入一个自定义的外部 ID”

File: docs/docs/workshop/workshop_guide.md (hunk 1 above)
Comment (dengmingtong) [reply docs/docs/workshop/workshop_guide.md:18]: Fixed

**Review by zxkane — APPROVED** [review]

**Pull request merged**